### DISCUSSION TITLE
Qt interface TODO before shipping

### DISCUSSION THREAD

**Discussion by stenzek** [PCSX2/pcsx2#5947] [General]
- [x] Can't do video recording
- [x] Add TV Shader options
- [x] internal screenshots
- [x] Can't make GS dumps
- [x] Adding more Pad types
- [x] Mouse analogue support
- [x] Guitar support
- [x] Adding USB
- [x] Expanding Audio - ???
- [x] Expanding Memcards - ??? - Include Multitap Memcard ports
- [x] Fix the labeler (moved around the files, gswx is unresponsive, etc)
- [x] When in fullscreen + Confirm On Shut Down + Pressing esc doesn't present in front of user
- [x] Emulog doesn't seem to be created in file-form
- [x] Changing Cubeb to XAudio2 while started in Cubeb first (hardcrash)
- [x] Set up translation workflow
- [x] Autoupdater (mostly done)
- [x] Add missing descriptions for settings
- [x] hiding mouse cursor
- [x] disable window resize isn't done
- [x] axis scale/vibration scale in controller UI
- [x] Disc drive selection in the UI
- [x] Controller profiles
- [x] Ability to mount an ISO then run an ELF file (for modding games)

**Comment by RedDevilus** [comment]
Haven't updated the list so Audio just need that small fix for crashing which is already on there and memcards is mostly fine.
But yeah people are going to want Escape doing pause and resume (some people think it kills the game but no it's in suspension)

**Reply by stenzek** [reply]
Then rebind it? Or change the defaults, so that pause is bound to escape. I don't really mind.

But that said, I was planning on binding escape to the ingame UI/menu when I brought it across.

**Comment by RedDevilus** [comment]
Also need to find out how to drag and drop ISO and other formats like ELFs and ask for boot.

**Comment by Blackbird88** [comment]
What about command line arguments? Last I tried them, none of them worked.

**Reply by refractionpcsx2** [reply]
Yes, they'll need doing also

**Reply by RedDevilus** [reply]
CLI reminder:
https://github.com/PCSX2/pcsx2/pull/3435
https://wiki.pcsx2.net/Command-line_support

**Comment by dio-gh** [comment]
Could perhaps promote this discussion into an issue ticket now.

**Comment by refractionpcsx2** [comment]
I'd prefer it stay separate from the noise.

**Comment by RedDevilus** [comment]
https://github.com/PCSX2/pcsx2-windows-dependencies/releases

Also need an easier/better way on how to handle compiling with Qt

**Reply by shikulja** [reply]
Determining covers by hash does not work.
Duck method (HASH-xxxxxx). I know it's early, I just hope they don't forget about it.
HASH-GAME must have a higher precedence than ID-GAME, this does not change anything in normal use, but is useful in some cases.
When there is a game with the same ID. But with different HASH and cover. For example, fan translations of the game with a different cover. Or game versions like demo-1.0-1.1

**Comment by RedDevilus** [comment]
Other issue that was discovered is that texture offsets and skipdraw ranges are not editable when in per-game config mode:
![image](https://user-images.githubusercontent.com/24227051/169720972-40f5e651-71ca-407e-92ab-5b8624b0d498.png)

**Comment by The-King-of-Toasters** [comment]
Does "Adding more Pad types" include better support for the DualShock 3? AFAIK SDL doesn't have an API for pressure-sensitivity.

**Comment by UserGit-2021** [comment]
Return function emulation right stick to mouse.

**Comment by refractionpcsx2** [comment]
Disc launching doesn't work currently, needs the device selection. Sten says we can use the popup from Duck  https://github.com/stenzek/duckstation/blob/master/src/duckstation-qt/mainwindow.cpp#L534-L574

**Comment by refractionpcsx2** [comment]
Hook up the TV shaders to the UI

**Comment by stenzek** [comment]
All of these have been completed.
